pycbc/inject/inject.py

```diff
@@ -637,6 +650,7 @@ def supported_approximants():
         for d in [waveform.waveform.td_wav, waveform.waveform.fd_wav]:
             for key in d:
                 all_apprxs.extend(d[key])
+        all_apprxs.extend(waveform.waveform.fd_det)
```

Comment:
Why is this a separate line?

Reply:
@ahnitz This needs to be a separate line. Because I follow `fd_det_squence`, just set `fd_det` as {}, not a `_scheme.ChooseBySchemeDict()` as `fd_wav`